.github/workflows/ci.yml

```
@@ -23,6 +23,8 @@ jobs:
             target
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
       - name: Build
-        run: cargo build --verbose
+        run: cargo build --all --verbose
       - name: Run tests
-        run: cargo test --verbose
+        run: cargo test --all --verbose
+      - name: Run tests as root
+        run: cargo test --all --no-run 2>&1 | grep "Executable" | tee /dev/stderr | awk '{ print $5 }' | tr -d '()' | xargs -I {} sudo sh -c "{}"
```

```suggestion
        run: cargo test --all --no-run 2>&1 | grep "Executable" | tee /dev/stderr | awk '{ print $4 }' | tr -d '()' | xargs -I {} sudo sh -c "{}"
```

I've tested this locally, `$5` was empty for me and `$4` had the executables...

---

Yep, same.
I'm not sure why CI is like this, but can definitely confirm it's running the way I want it to in gh actions.